template-component/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "trailingComma": "es5"
+}
```

Comment:
happy to snip - this was before I decided to just use the VSCode default (es5) instead of the prettier default (all). no longer seeing issues in folks' component repos